packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-listener-rule.ts

```diff
@@ -100,6 +108,40 @@ export interface FixedResponse {
   readonly messageBody?: string;
 }
 
+/**
+ * A redirect response
+ */
+export interface RedirectResponse {
+  /**
+   * The hostname. This component is not percent-encoded. The hostname can contain #{host}.
```

Comment:
This value is optional. Please describe the default using a `@default` tag. This comment holds for all fields in this struct.